projects/packages/videopress/src/client/admin/components/video-card/index.tsx

```diff
@@ -62,6 +63,12 @@ const ProcessingThumbnail = () => (
 	</div>
 );
 
+const PrivateThumbnail = () => (
+	<div className={ styles[ 'video-card__custom-thumbnail' ] }>
+		<Icon icon={ captureVideo } size={ 96 } />
+	</div>
+);
```

Comment:
I wonder whether creating a new component does make sense, especially considering that you are not taking advantage of the CSS class. I guess it was part of your development flow which is fine. Just consider removing it and using straight `<Icon icon={ captureVideo } size={ 96 } />`. BTW it looks pretty nice.

<img width="990" alt="image" src="https://user-images.githubusercontent.com/77539/196303022-d5914e3a-eea9-4bd9-a485-146f4c89dd9f.png">

Reply:
@retrofox This is what I get when using only the icon:

![image](https://user-images.githubusercontent.com/6760046/196494515-b9e87b28-f7b0-4312-8688-2c17455e0686.png)

Seems like I do need this `<div className={ styles[ 'video-card__custom-thumbnail' ] }>` wrapper to make the style works properly.

Is it OK to use it inline to prevent the component creation? Like this:

```
	thumbnail = privacyIsPrivate ? (
		<div className={ styles[ 'video-card__custom-thumbnail' ] }>
			<Icon icon={ captureVideo } size={ 96 } />
		</div>
	) : thumbnail;
```